cli/azd/pkg/azdo/build_policy.go

```diff
@@ -97,3 +143,68 @@ func CreateBuildPolicy(
 
 	return nil
 }
+
+func newLocalClient(ctx context.Context, connection *azuredevops.Connection) (*clientImpl, error) {
```

Comment:
Just making sure I understand what's going on with this `localClient` thing?  In the future we would be able to get rid of all this stuff (and just use an `azuredevops.Client` or some other type from that pacakge?) assuming that the linked issue is fixed and we can pass a continuation token?